src/platforms/javascript/guides/cordova/upload-debug.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Debug Information
+sidebar_order: 2
+description: "Learn more about uploading debug symbols when building your app."
+---
+
+Debug symbols provide information that Sentry displays on the Issue Details page to help you triage an issue.
+
+<Note>
+
+We do not currently support automatic source map uploads or build scripts for Android. You must upload files manually [using our CLI](/product/cli/dif/#uploading-files)."
```

Comment:
```suggestion
We do not currently support non-javascript automatic source map uploads or build scripts for Android. You must upload files manually [using our CLI](/product/cli/dif/#uploading-files)."
```
Maybe we can make it clear here that this is referring only to non-javascript debug files?